service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -90,9 +93,9 @@
         super("cmo-progression");
     }
 
-    @BeforeEach
+    @BeforeAll
```

Comment:
Doesn't `@BeforeAll` require a static method?

Reply:
I couldn't make it a static method as now() is non-static. Added @TestInstance(PER_CLASS) which allows @BeforeAll on non-static.

Reply:
I would say it may be better to go with simplicity then and leave it as the `@BeforeEach` then, thoughts?

Reply:
Ok I will change back to @BeforeEach.